src/components/Header/Header.module.scss

```diff
@@ -1,6 +1,7 @@
 .header {
   padding: 3rem 2rem;
   color: white;
+  background-color: #111 !important;
```

Comment:
Can we avoid `!important`, why is it necessary? Be more specific while targeting if needed. It's not an issue now, but it could bite us in the future